### PR TITLE
Change error count messages

### PR DESCRIPTION
**Pull request by citizen428**
See #33525 for details. r? @Mark-Simulacrum

### CODE REVIEW

File: src/librustc_errors/lib.rs [hunk 1]
```diff
@@ -501,9 +501,8 @@ impl Handler {
 
                 return;
             }
-            1 => s = "aborting due to previous error".to_string(),
             _ => {
-                s = format!("aborting due to {} previous errors", self.err_count.get());
+                s = format!("aborting due to previous error(s)".to_string);
```
Comment (Mark-Simulacrum) [inline src/librustc_errors/lib.rs]: The call to `to_string` should just be removed (or made into a call and the `format!` removed.

**Comment by arielb1** [comment]
Looks like you need to update the UI tests:
```
[01:04:45] ---- [ui] ui/suggestions/confuse-field-and-method/private-field.rs stdout ----
[01:04:45] 	ui: /checkout/src/test/ui/suggestions/confuse-field-and-method/private-field.rs
[01:04:45] normalized stderr:
[01:04:45] error: no method named `dog_age` found for type `animal::Dog` in the current scope
[01:04:45]   --> $DIR/private-field.rs:26:23
[01:04:45]    |
[01:04:45] 26 |     let dog_age = dog.dog_age();
[01:04:45]    |                       ^^^^^^^ private field, not a method
[01:04:45] 
[01:04:45] error: aborting due to previous error(s)
[01:04:45] 
[01:04:45] 
[01:04:45] 
[01:04:45] expected stderr:
[01:04:45] error: no method named `dog_age` found for type `animal::Dog` in the current scope
[01:04:45]   --> $DIR/private-field.rs:26:23
[01:04:45]    |
[01:04:45] 26 |     let dog_age = dog.dog_age();
[01:04:45]    |                       ^^^^^^^ private field, not a method
[01:04:45] 
[01:04:45] error: aborting due to previous error
[01:04:45] 
[01:04:45] 
[01:04:45] 
[01:04:45] diff of stderr:
[01:04:45] 
[01:04:45]  error: no method named `dog_age` found for type `animal::Dog` in the current scope
[01:04:45]    --> $DIR/private-field.rs:26:23
[01:04:45]     |
[01:04:45]  26 |     let dog_age = dog.dog_age();
[01:04:45]     |                       ^^^^^^^ private field, not a method
[01:04:45]  
[01:04:45] -error: aborting due to previous error
[01:04:45] +error: aborting due to previous error(s)
```

There's some command that updates the tests automatically, but I don't remember what it is. @Mark-Simulacrum do you remember how to run it?

**Comment by citizen428** [comment]
@arielb I was wondering if there's a command, the ones I already updated I used `sed` for. Clearly I missed some, will look into it.

**Comment by Mark-Simulacrum** [comment]
Wasn't able to find or get someone to answer me about the potential script, but looks like Travis failed with on `ui/codemap_tests/two_files.rs`.

**Comment by citizen428** [comment]
@Mark-Simulacrum Yes, I fixed that already, that's the build that's currently running (I had a previous commit where I fixed almost all of the others, but this one slipped through). Fingers crossed this will finally be green. I just wish tests would run faster on my local machine, I feel bad always tying uo Travis.

**Comment by citizen428** [comment]
Ok, this is getting ridiculous, I had amended and force pushed an old checkout (switched from home to work computer).... I'll need to take some time to fix this again, sorry for all the noise 🤦‍♂️

**Comment by Mark-Simulacrum** [comment]
No problem! Just to check, in case you weren't aware, `./x.py test -i --stage 1 src/test/ui` is probably the best way to run them locally today.

@bors r+

**Pull request merged**